src/io/flutter/android/AndroidEmulator.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.android;
+
+import com.intellij.execution.ExecutionException;
+import com.intellij.execution.configurations.GeneralCommandLine;
+import com.intellij.execution.process.OSProcessHandler;
+import com.intellij.execution.process.ProcessAdapter;
+import com.intellij.execution.process.ProcessEvent;
+import com.intellij.execution.process.ProcessOutputTypes;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.util.Key;
+import com.intellij.openapi.vfs.VirtualFile;
+import io.flutter.FlutterMessages;
+
+public class AndroidEmulator {
+  private static final Logger LOG = Logger.getInstance(AndroidEmulator.class);
+
+  final AndroidSdk androidSdk;
```

Comment:
That's probably my fault. I didn't follow JetBrain's my prefix convention, since I try to avoid common prefixes and nobody else does that.  (So many standards to choose from...)

Rationale: if you have both common prefixes and common suffixes, names become longer and harder to distinguish because you have to look at a few characters in the middle of the identifier. Of the two, I'd rather have common suffixes and distinct prefixes.

(But at least we're not mixing conventions in a single file.)